### PR TITLE
Test standby activate with negative cli parameters

### PR DESCRIPTION
**Pull request by pdebski21**
Signed-off-by: Piotr Debski <piotr.debski@intel.com>

### CODE REVIEW

File: test/functional/tests/cli/test_cli_standby.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        cache_id = 1
+        cache_line_size = 32
+
+    with TestRun.step("Init standby cache"):
+        cmd_init = casadm_bin + f"--standby --init --cache-id {cache_id} " \
+                                f"--cache-device {cache_device.path} --cache-line-size" \
+                                f" {cache_line_size} --force"
+        output = TestRun.executor.run(cmd_init)
+        if output.exit_code == 0:
+            TestRun.LOGGER.error(f'"Standby activate failed with command {cmd_init}"')
```
Comment (mmichal10) [inline test/functional/tests/cli/test_cli_standby.py:39]: Consider using `standby_init()` function from `api/cas/casadm.py`. It wraps setting up a command and handling error. Additionally it returns `Cache` object.

File: test/functional/tests/cli/test_cli_standby.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+    with TestRun.step("Detach standby cache"):
+        cmd_init = casadm_bin + f"--standby --detach --cache-id {cache_id}"
+        output = TestRun.executor.run(cmd_init)
+        if output.exit_code == 0:
+            TestRun.LOGGER.error(f'"Standby activate failed with command {cmd_init}"')
```
Comment (mmichal10) [inline test/functional/tests/cli/test_cli_standby.py:45]: How about adding `standby_detach()` method to `Cache` class?

File: test/functional/api/cas/cache.py [hunk 3]
```diff
@@ -173,3 +173,6 @@ def get_cache_config(self):
         return CacheConfig(self.get_cache_line_size(),
                            self.get_cache_mode(),
                            self.get_cleaning_policy())
+
+    def standby_detach(self, shortcut: bool = False):
+        casadm.standby_detach_cache(cache_id=self.cache_id, shortcut=shortcut)
```
Comment (mmichal10) [inline test/functional/api/cas/cache.py:178]: `casadm.standby_detach_cache()` returns an object with result, stdout and stderr. You need to pass it to the caller.

Comment (pdebski21) [reply test/functional/api/cas/cache.py]: Adding return will be enaugh? I don't see value in using it in this specific test.

Comment (mmichal10) [reply test/functional/api/cas/cache.py]: I agree it is unnecessary in this test, but in general API should be consistent

**Review by mmichal10 — APPROVED** [review]

**Pull request merged**